### PR TITLE
Updated to support Django 1.7's ability to use classes as entries of INS...

### PR DESCRIPTION
**Pull request by hekevintran**
...TALLED_APPS

Solution for #68

### CODE REVIEW

File: jingo/__init__.py [hunk 1]
```diff
@@ -49,7 +50,19 @@ def get_env():
     # TEMPLATE_DIRS and packages in INSTALLED_APPS.
     x = ((jinja2.FileSystemLoader, settings.TEMPLATE_DIRS),
          (jinja2.PackageLoader, settings.INSTALLED_APPS))
-    loaders = [loader(p) for loader, places in x for p in places]
+    loaders = []
```
Comment (jezdez) [inline jingo/__init__.py]: This should use the recommended way to iterate over the list of app config instances, `for app_config in apps.get_app_configs()`. So just pass in `apps.get_app_configs` as the option to the `PackageLoader` if Django's version is equal or higher than 1.7. Then iterate over the list of app configs and get their path attribute for the dotted import path.

Comment (jsocol) [inline jingo/__init__.py]: @jezdez any chance you have a minute to apply that change to this patch? i'm running into it with jingo + adminplus and i'm not really sure I understand that comment

Comment (jsocol) [inline jingo/__init__.py]: maybe we should cut a 0.8 and drop dj1.6

**Comment by jsocol** [comment]
Fixed by #74. Thanks for getting this started!

**Pull request closed without merging**